python/paddle/fluid/layers/nn.py

```diff
@@ -4607,7 +4612,14 @@ def hsigmoid(input,
             and :math:`D` is the feature size.
         label (Variable): The tensor variable contains labels of training data.
             It's a tensor with shape is :math:`[N \\times 1]`.
-        num_classes: (int), The number of classes, must not be less than 2.
+        num_classes: (int), The number of classes, must not be less than 2. with default tree this has to be set
```

Comment:
What happen if num_classes is None. Should doc

Reply:
changed, and num_class should never be None